tools/copyblocks/main.go

```diff
@@ -59,6 +62,7 @@ func (c *config) registerFlags(f *flag.FlagSet) {
 	f.DurationVar(&c.copyPeriod, "copy-period", 0, "How often to repeat the copy. If set to 0, copy is done once, and the program stops. Otherwise, the program keeps running and copying blocks until it is terminated.")
 	f.Var(&c.enabledUsers, "enabled-users", "If not empty, only blocks for these users are copied.")
 	f.Var(&c.disabledUsers, "disabled-users", "If not empty, blocks for these users are not copied.")
+	f.Var(&c.crossTenantMapping, "cross-tenant-mapping", "A comma-separated list of (source tenant):(destination tenant). If a source tenant is not mapped then its destination tenant is assumed to be identical.")
```

Comment:
> cross-tenant-mapping

(nit) Maybe it's better to stick to "users" on the inputs level? The command has `--enabled-users` and `--disabled-users` flags. So it feels confusing if one doesn't know that those are the same. Can we name the flag `--user-mapping`?

Reply:
Nice catch, thanks. I changed it to `--user-mapping` and updated the documentation to reflect that.